components/library/gamin/Makefile

```diff
@@ -44,12 +41,16 @@ COMPONENT_PREP_ACTION= ( cd $(@D)  && \
 	automake -c -f -a &&\
 	autoconf )
 
+PYV += `echo $(PYTHON_VERSION) | tr -d .`
+COMPONENT_POST_INSTALL_ACTION.64 += ( cd $(PROTO_DIR)/$(PYTHON_VENDOR_PACKAGES) && \
+					$(MV) _gamin.so _gamin.cpython-$(PYV).so )
```

Comment:
To have this future proof you should ask Python for `EXT_SUFFIX`.  You could look at `python/compizconfig-python/Makefile` for an example.